### DISCUSSION TITLE
Any demos using htmx-spring-boot-thymeleaf?

### DISCUSSION THREAD

**Discussion by LiveNathan** [wimdeblauwe/htmx-spring-boot#60] [Q&A] [answered]
Are there any demos out there using htmx-spring-boot-thymeleaf? I have seen the TodoMVC with Thymeleaf and HTMX on Wim's site, but it is not using htmx-spring-boot-thymeleaf, as far as I can tell.

**Comment by wimdeblauwe** [comment, marked as answer]
There is also https://www.wimdeblauwe.com/blog/2023/02/20/toasts-notifications-in-thymeleaf-with-shoelace-and-htmx/ and https://www.wimdeblauwe.com/blog/2022/06/15/thymeleaf-and-htmx-with-out-of-band-swaps/.

I am also working on a book about Thymeleaf and htmx that will have many examples and demo's, which is due to be released towards the end of the year.

**Reply by LiveNathan** [reply]
Thanks!
